docs: adding more documentation about GCP resources (#53)
lacework/terraform-provider-lacework: website/docs/r/integration_gcp_at.html.markdown

}
```

## GCP Audit Trail Module Usage

> Lacework maintains a Terraform module that can be used to create and manage the necessary
> resources required from both, the cloud provider platform as well as the Lacework platform.

Author: ```suggestion
Lacework maintains a Terraform module that can be used to create and manage the necessary
resources required for both, the cloud provider platform and the Lacework platform.
```

> Lacework maintains a Terraform module that can be used to create and manage the necessary
> resources required from both, the cloud provider platform as well as the Lacework platform.

Author: ```suggestion
Lacework maintains a Terraform module that can be used to create and manage the necessary
resources required for both, the cloud provider platform as well as the Lacework platform.
```
